core/embed/rust/src/ui/layout_eckhart/component/button.rs

```diff
@@ -624,10 +652,32 @@ impl Component for Button {
 
     fn place(&mut self, bounds: Rect) -> Rect {
         self.area = bounds;
+        let subtext_start =
+            self.baseline_text_height() * 2 + constant::LINE_SPACE + self.baseline_subtext_height();
+        self.subtext_marquee.set_y_offset(subtext_start);
+        self.subtext_marquee.place(self.area.inset(Insets::new(
+            0,
+            self.content_offset.x,
+            0,
+            self.content_offset.x / 2,
```

Comment:
It's questionable, whether the initial position should be aligned with the main text or whether the subtext should not go more left than the main text. Personally, I would be for the second one but haven't strong opinion about it.
```suggestion
            self.content_offset.x,
```